packages/notification/src/ToastNotification.tsx

```diff
@@ -18,7 +18,7 @@ type ToastNotificationProps = Omit<
 const ToastNotification = ({
   id,
   hideCloseIcon = false,
-  autohide = true,
+  autohide = false,
   autohideDelay = "short",
```

Comment:
Might as well change the default on this while we're here too

```suggestion
  autohideDelay = "long",
```